src/explore-education-statistics-admin/src/pages/release/pre-release/components/PublicPreReleaseAccessForm.tsx

```diff
@@ -88,9 +88,9 @@ const PublicPreReleaseAccessForm = ({
         </Formik>
       ) : (
         <>
-          {isReleaseLive && (
+          {!canUpdateRelease && (
```

Comment:
Just wondering if the check around the inset text above should also be changed from `!isReleaseLive` to `!canUpdateRelease`? (L51 - Github won't let me leave a comment on the line itself)

Reply:
I'm not 100% sure - I was happy changing it here as this `This release has been approved, and can no longer be updated.` is always controlled by the `canUpdateRelease` flag on all other pages, whereas the logic above is governed by slightly different rules.  So I'd be tempted to leave that as is for now as it seems to be deliberately tied to the "liveness" of the Release.

Reply:
Ok, makes sense 👍